rest_framework/urlpatterns.py

```diff
@@ -9,7 +9,7 @@ def _get_format_path_converter(suffix_kwarg, allowed):
         if len(allowed) == 1:
             allowed_pattern = allowed[0]
         else:
-            allowed_pattern = '(?:%s)' % '|'.join(allowed)
+            allowed_pattern = f'(?:{"|".join(allowed)})'
```

Comment:
This is one where it starts to look a bit confusing to me.

Perhaps we ought to pull the `"|".join(allowed)` out of the f-string?...

```suggestion
            allowed_cases = "|".join(allowed)
            allowed_pattern = f'(?:{allowed_cases})'
```